src/package.rs

```diff
@@ -42,10 +42,23 @@ pub mod rpm_parsing {
                 None
             };
 
+            /*
+            ogriginal source: https://github.com/rpm-software-management/createrepo_c/blob/70e92f6a802059f1f8d003299cea925e838745b1/src/parsehdr.c#L425
+            // Calculate pre value
+                    if (num_flags & (RPMSENSE_PREREQ |
+                                     RPMSENSE_SCRIPT_PRE |
+                                     RPMSENSE_POSTTRANS |
+                                     RPMSENSE_PRETRANS |
+                                     RPMSENSE_SCRIPT_POST))
+                    {
+                        pre = 1;
+                    }
+             */
```

Comment:
I'd prefer not to copy and paste code, or even link to it necessarily, to keep clear of licensing issues.